### PR TITLE
Update applet_installation.md

### PR DESCRIPTION
**Pull request by gravityblast**

### CODE REVIEW

File: source/keycard_api/applet_installation.md [hunk 1]
```diff
@@ -5,13 +5,7 @@ title: Getting Started
...
-
-The installer has a simple interface, when you open it, you see a white screen and a few buttons. Select the operation you want to perform, then bring the card near to the NFC reader of your phone and wait until the operation is complete. Make sure NFC is turned on.
-
-There are two operations used to install the applet. One is "Install Secure", which installs the applet without sending the INIT command, so the installed applet is not initialized. The other option is "Install with test PIN/PUK". This installs the applet and sets PIN to "000000", PUK to "123456789012" and Pairing Password to "KeycardTest". This second option is useful if you want to quickly test interaction without having to initialize the card in your application.
+If you got a Keycard and you are integrating it in your project, you will most likely want to reset the applet status completely for testing purposes. This is done by reinstalling the applet, which can be accomplished using the [Keycard CLI](https://github.com/status-im/keycard-cli). To use  the Keycard CLI, a USB reader is needed to be able to communicate with the card.
```
Comment (bitgamma) [inline source/keycard_api/applet_installation.md]: maybe you could add some directions on how to install the card with the cli tool both in pre-INIT and post-INIT state?

Comment (jakubgs) [reply source/keycard_api/applet_installation.md]: @gravityblast any intention of addressing this comment?

Comment (gravityblast) [reply source/keycard_api/applet_installation.md]: @bitgamma @jakubgs  sorry I missed the comment.
@bitgamma it's already explained in the link, do you think it's better to put it here as well?

Comment (jakubgs) [reply source/keycard_api/applet_installation.md]: fair enough

**Pull request merged**